src/admin/settings/settings.html

```diff
@@ -0,0 +1,83 @@
+<legend class="frame-title">{{'admin.platform.settings' | translate}}</legend>
+<form id="settingsFileForm" enctype="multipart/form-data" ng-submit="uploadSettings()">
+    <fieldset>
+        <h4 class="section-title">{{'admin.settings.upload' | translate}}</h4>
+        <div class="form-group">
+            <label for="fileInput">{{'admin.settings.fileChosen' | translate}}</label>
+            <input class="button-file" type="file" id="fileInput" name="file" value="{{'server.button.selectFile' | translate}}" admin-import="settingsFile"/>
+        </div>
+    </fieldset>
+    <input type="submit" value="{{'admin.submit' | translate}}" />
+</form>
+<form id="settingsLocationForm" method="post" ng-submit="uploadFileLocation()">
+    <fieldset>
+        <h4 class="section-title">{{'admin.settings.location' | translate}}</h4>
+        <div class="form-group">
+            <label for="settingsLocation">{{'admin.settings.location.new' | translate}}</label>
+            <input type="text" id="settingsLocation" name="settingsLocation" ng-model="location"/>
+        </div>
+    </fieldset>
+    <input type="submit" value="{{'admin.submit' | translate}}"/>
+</form>
+<form>
+    <fieldset>
+        <h4 class="section-title">{{'admin.settings.importexport.header' | translate}}</h4>
+        <div class="form-group">
+            <label for="exportConfig">{{'admin.settings.importexport.export' | translate}}</label>
+            <input class="button button-export" type="button" id="exportConfig" name="exportConfig" ng-click="exportConfig()" value="{{'admin.settings.importexport.exportButton' | translate}}" />
+        </div>
+    </fieldset>
+</form>
+
+<h4 class="section-title">{{'admin.settings.platform' | translate}}</h4>
+<div ng-repeat="pSettings in platformSettings.settingsList">
+    <form id="platformSettingsForm">
```

Comment:
@mkruszynski Is there a reason the ng-repeat isn't on the form element? This is a super minor thing, but it would be nice to have as few div's as possible